app/assets/stylesheets/forms/_inputs.scss

```diff
@@ -92,15 +92,15 @@ input[type=radio] {
 			}
 
 			+ label {
-				@media (min-width: 500px) {
-					font-size: 1.2em;
+				@media (min-width: 31.25rem) {
```

Comment:
also concerned about media query breakpoints in rem...

Reply:
kind of touched on this on another comment. lmk if you want me to reiterate it here.